sdk/js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@devcycle/devcycle-js-sdk",
-  "version": "1.2.15-alpha.0",
+  "version": "1.2.15",
```

Comment:
do not change, this can stay as is with the alpha tag.. otherwise this will publish a new version of the React SDK with version `1.2.15`